test/run

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess
+import sys
+
+
+# FIXME: Learn image-prepare to read image from TEST_OS
+def prepare(prefix, value, image):
+    sys.stderr.write("Preparing image: building and installing Cockpit ...\n")
+
+    try:
+        # Download all the additional images so that even older branches find them
+        subprocess.check_call([os.path.join("bots", "image-download"),
+                               "candlepin", "fedora-stock", "fedora-23-stock",
+                               "ipa", "openshift", "selenium"])
+
+    except subprocess.CalledProcessError:
+        return "Downloading of additional images failed"
```

Comment:
Let's please drop this now. It's annoying enough locally, and hasn't been necessary with our current CI for ages.

Reply:
What do you mean? Downloading of those additional images?

Reply:
Yes, the entire paragraph.